_posts/2024-05-14-homebrew-4.3.0.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 4.3.0
+author: MikeMcQuaid
+redirect_from: /blog/4.3.0/
+---
+
+Today, I'd like to announce Homebrew 4.3.0.
+The most significant changes since 4.2.0 are SBOM support, initial attestation verification, new command analytics and uninstall autoremove by default.
+
+Major changes and deprecations since 4.2.0:
+
+- [`brew bottle` will include a basic SPDX file inside the bottle](https://github.com/Homebrew/brew/pull/16594)
+  [and a more comprehensive one after installation](https://github.com/Homebrew/brew/pull/17254).
+  This is to provide support for the widely used SBOM format from Homebrew.
+
+- [If `HOMEBREW_VERIFY_ATTESTATIONS` is set, `brew install` will verify the attestation chain of trust (wording?) when pouring bottles using GitHub's `gh` CLI](https://github.com/Homebrew/brew/pull/17049).
+  This functionality is still in beta, we expect to remove the need for the `gh` tool and improve performance before we make this the default behaviour.
+  This behaviour demonstrates Homebrew's ongoing commitment to improving our security posture.
+  Read more in [the tracking issue](https://github.com/Homebrew/brew/issues/17019).
+
+- [`HOMEBREW_AUTOREMOVE` is the default behaviour meaning that `brew cleanup` and `brew uninstall` automatically run `brew autoremove`](https://github.com/Homebrew/brew/pull/17261).
+   Disable this by setting `HOMEBREW_NO_AUTOREMOVE`.
+   This is to improve the default behaviour of `brew uninstall` given `brew autoremove` is sufficiently reliable.
+
+- [Homebrew has two new types of analytics](https://github.com/Homebrew/brew/pull/16847): ["Brew Command Run" events](https://formulae.brew.sh/analytics/brew-command-run/30d/) and `brew test-bot` analytics.
+  The latter are not working or published yet but will be soon.
+  These are to help us improve the documentation and prioritisation of issues in Homebrew.
+
+- [Homebrew/homebrew-cask requires code signing of all casks](https://github.com/Homebrew/brew/pull/17002).
+  Expect removal of casks that are not code signed from Homebrew/homebrew-cask in future.
+  This is because code signing is required on Apple Silicon which is a growing proportion of all Homebrew users.
```

Comment:
```suggestion
  This is because code signing is required on Apple Silicon which is used by [a growing majority of all Homebrew users](https://formulae.brew.sh/analytics/homebrew-os-arch-ci/365d/).
```